libs/sdk-ui-ext/src/internal/utils/dimensions.ts

```diff
@@ -37,9 +39,16 @@ export function getPivotTableDimensions(insight: IInsightDefinition): IDimension
     const rowTotals = safeBucketTotals(insight, BucketNames.ATTRIBUTE);
     const colTotals = safeBucketTotals(insight, BucketNames.COLUMNS);
 
+    const measureGroupDimension: string = getMeasureGroupDimensionFromProperties(
+        insightProperties(insight),
+    ) ?? "columns";
```

Comment:
Maybe we should have some constants, like we have with BucketNames.ATTRIBUTE (something like `MeasureGroupDimension.COLUMNS`, `MeasureGroupDimension.ROWS` perhaps?) And use them not only here but also later in PivotTable.tsx (similar logic is there).

Reply:
UI SDK is not fan of constants, enum types are used more often and we rely only on typescript with value validation, eg. 
export type ValidationContext = "CORS" | "UI_EVENT" | "DRILL_TO_URI";

whole libs/api-client-bear/src/interfaces.ts etc